### DISCUSSION TITLE
pulseaudio install not working

### DISCUSSION THREAD

**Discussion by scottharvey** [Homebrew/discussions#523] [Everyday usage]
I have run `brew upgrade` and I'm not using `Homebrew 2.7.3`

I installed pulseaudio using `brew install pulseaudio`

Then when I try to run `pulseaudio` I get the following error

```
> pulseaudio
W: [] caps.c: Normally all extra capabilities would be dropped now, but that's impossible because PulseAudio was built without capabilities support.
```

There seems to be a few old tickets that had the same issue but they are closed now https://github.com/Homebrew/homebrew-core/issues/30062 https://github.com/Homebrew/homebrew-core/issues/9314

Can anyone recreate this issue or is it just me?

**Comment by Azavard** [comment]
I'm having the same issue, with the following erros:

> W: [] caps.c: Normally all extra capabilities would be dropped now, but that's impossible because PulseAudio was built without capabilities support.
E: [] pid.c: Daemon already running.
E: [] main.c: pa_pid_file_create() failed.
